Sources/PostgresNIO/New/Connection State Machine/ExtendedQueryStateMachine.swift

```diff
@@ -480,6 +597,16 @@ struct ExtendedQueryStateMachine {
             preconditionFailure("Invalid state")
         }
     }
+
+    mutating func channelWritabilityChanged(isWritable: Bool) {
+        guard case .copyingData(.pendingBackpressureRelieve(let continuation)) = state else {
+            return
+        }
+        self.avoidingStateMachineCoW { state in
+            state = .copyingData(.readyToSend)
+            continuation.resume()
```

Comment:
side effect triggered in state machine. return the continuation out and succeed it in the PostgresChannelHandler.